src/main/kotlin/pl/piomin/services/controller/PersonController.kt

```diff
@@ -1,16 +1,12 @@
 package pl.piomin.services.controller
 
-import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.web.bind.annotation.*
 import pl.piomin.services.model.Person
 import pl.piomin.services.repository.PersonRepository
 
 @RestController
 @RequestMapping("/persons")
-class PersonController {
-
-    @Autowired
-    lateinit var repository: PersonRepository
+class PersonController(val repository: PersonRepository) {
```

Comment:
Well it's ok, but I wanted to show @Autowired with lateinit in my article / sample